cmd/argocd/commands/app.go

```diff
@@ -2121,3 +2125,144 @@ func NewApplicationPatchResourceCommand(clientOpts *argocdclient.ClientOptions)
 
 	return command
 }
+
+// used to format the output of when tailing pod logs
+type podLogFormatter struct{}
+
+// takes a log entry from PodLogs(), and makes it readable
+func (plf *podLogFormatter) Format(entry *logrus.Entry) ([]byte, error) {
```

Comment:
This replaces the `logrus-nested-formatter` and makes it easy for humans to read. I wasn't sure _where_ this should go, `app.go` feels weird to put it here.